### DISCUSSION TITLE
TEXT와 VARCHAR의 차이점은 무엇일까?

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#41] [General]
# TEXT와 VARCHAR의 차이점은 무엇일까?

Scan @Component @Repository  는 repository(DAO로 알려진) 의 역할을 하는 class에 붙인다.
 @Repository  를 사용하면 예외의 자동변환을 가능하게 해준다. (Exception Translation) S…

https://cyma-s.github.io/text-vs-varchar/

**Comment by kevstevie** [comment]
낚시글 쓰지마세요

**Reply by Cyma-s** [reply]
월척이네요~?
